inductiva/templates/heat_sink/openfoam/files/system/fvSchemes

```diff
@@ -0,0 +1,30 @@
+/*--------------------------------*- C++ -*----------------------------------*\
+  =========                 |
+  \\      /  F ield         | OpenFOAM: The Open Source CFD Toolbox
+   \\    /   O peration     | Website:  https://openfoam.org
+    \\  /    A nd           | Version:  7
+     \\/     M anipulation  |
+\*---------------------------------------------------------------------------*/
+FoamFile
+{
+    version     2.0;
+    format      ascii;
+    class       dictionary;
+    location    "system";
```

Comment:
Is this doing anything?

Reply:
I think it must exist. The analogous file for each region of the domain (fluid/heater/fins) then overrides whatever is set here.